### ISSUE TITLE
Address auto-population not working properly in IE11/Edge

### ISSUE THREAD

**Issue by jwrudzin** [verbb/formie#317]
**Description**
Address auto-population is not working in IE11 or Edge.

**Steps to reproduce**
1. Create a form with an address auto-population field using Google Places.
2. Using IE11 or Edge, start typing an address. No auto-population suggestions appear.

### Form settings
- Multi-page form: No
- Submission Method: Page Reload
- Client-side Validation: Yes
- Custom Form Templates: No

**Additional info**
- Plugin version: 1.3.13
- Craft version: 3.5.18
- Multi-site: No

**Comment by engram-design** [comment]
Should be fixed in the next release. To get the fix early, change your `verbb/formie` requirement in `composer.json` to:

```
"require": {
  "verbb/formie": "dev-craft-3 as 1.3.13",
  "...": "..."
}
```

Then run `composer update`.

**Comment by engram-design** [comment]
Fixed in [1.3.14](https://github.com/verbb/formie/releases/tag/1.3.14)
